### PR TITLE
Bottom nav

### LINKED ISSUE #9: Create BottomNavBar
Add Icons with States/Props

### PR DESCRIPTION
**Pull request by boldCM**

### CODE REVIEW

File: client/src/components/BottomNav/BottomNav.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  margin-bottom: 9px;
+  position: fixed;
+  z-index: 1;
+`;
+
+const BottomNav = () => {
+  return (
+    <>
+      <Triangle></Triangle>
+      <BorderTraingle></BorderTraingle>
```
Comment (kschmitz123) [inline client/src/components/BottomNav/BottomNav.js]: i'm not 100% sure but you probably could change them to void elements?
```suggestion
      <Triangle/>
      <BorderTraingle/>
```

**Comment by kschmitz123** [comment]
lgtm 🚀

**Review by arassaria — APPROVED** [review]
LGTM 🚀

File: client/src/components/BottomNav/BottomNav.js (hunk 1 above)
Comment (boldCM) [reply client/src/components/BottomNav/BottomNav.js]: yesss, thank you! :)

**Review by Naasgul — APPROVED** [review]
LGTM! 👍

**Pull request merged**